obal/data/playbooks/update.yml

```diff
@@ -55,3 +48,27 @@
       with_items: "{{ new_source_urls.stdout_lines | list }}"
       when:
         - "'://' not in item"
+
+    - name: get spec file
```

Comment:
In other places (above) we use `*.spec` in tools. Should we move this part to the top and use the explicit variable?